### ISSUE TITLE
Version info in manifest.json

### ISSUE THREAD

**Issue by Cees-van-Beek** [Bouni/luxtronik#19]
I recent saw the following comment in the logfile of Home Assistant:

```
2021-03-12 23:12:31 WARNING (MainThread) [homeassistant.loader] No 'version' key in the manifest file for custom integration 'luxtronik'. This will not be allowed in a future version of Home Assistant. Please report this to the maintainer of 'luxtronik'
```
 I propose that @Bouni fix this.
Otherwise I can make a branche and make a pull request.

**Comment by Bouni** [comment]
I'll fix it, thanks for pointing this out!

**Comment by Bouni** [comment]
Fixed with release 0.1.17
